src/bp/ui-studio/src/web/actions/index.ts

```diff
@@ -489,3 +489,11 @@ export const getModuleTranslations = () => dispatch => {
     dispatch(receiveModuleTranslations(data))
   })
 }
+
+export const botsReceived = createAction('BOTS/RECEIVED')
+export const fetchBotIds = () => dispatch => {
+  // tslint:disable-next-line: no-floating-promises
```

Comment:
You can fix that error by just adding the .catch() after the .then() instead of escaping the tslint

Reply:
We're doing this everywhere in this file, if there's an error you want to know about it, not silent catch it

Reply:
I feel like we should catch it and handle it instead 🤔 anyways it's true that it's used everywhere, it could be something to talk about in the future (catching and handling the errors in a graceful and consistant way)